src/parser/paladin/retribution/modules/core/azeritetraits/DivineRight.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import {formatNumber, formatPercentage} from 'common/format';
+import {calculateAzeriteEffects} from 'common/stats';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+
+
+const divineRightStats = traits => Object.values(traits).reduce((obj, rank) => {
+  const [strength] = calculateAzeriteEffects(SPELLS.DIVINE_RIGHT.id, rank);
+  obj.strength += strength;
+  return obj;
+}, {
+  strength: 0,
+});
+
+export const STAT_TRACKER = {
+  strength: combatant => divineRightStats(combatant.traitsBySpellId[SPELLS.DIVINE_RIGHT.id]).strength,
+};
+
+/**
+ * When Divine Storm damages an enemy who is below 20% health, your Strength is increased by 419 for 15 sec.
+ *
+ * Example report: https://www.warcraftlogs.com/reports/yMVXFfhACTwJzKYp#fight=25
+ */
+class DivineRight extends Analyzer{
+  strength = 0;
+
+  constructor(...args){
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.DIVINE_RIGHT.id);
+    if(!this.active){
+      return;
+    }
+    const {strength} = divineRightStats(this.selectedCombatant.traitsBySpellId[SPELLS.DIVINE_RIGHT.id]);
```

Comment:
these two lines highlight the issue with returning an object from `divineRightStats`. I know this is how its done in other modules, but that is a mixture of (my) mistakes and other traits returning multiple stats.